components/applications/email-validator/index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Email validator
+permalink: /components/applications/email-validator/
+redirect_from:
+    - /applications/validator/
```

Comment:
Please don't use redirect_from unless they were previously in use. It just clutters the sitemap